lib/ipmi_backend_utils.pm

```diff
@@ -358,8 +358,9 @@ sub set_grub_on_vh {
 #ipmitool to perform server management
 sub ipmitool {
     my ($cmd) = @_;
+    my $ipmi_options = $bmwqemu::vars{IPMI_OPTIONS} // '-I lanplus';
 
-    my @cmd = ('ipmitool', '-I', 'lanplus', '-H', $bmwqemu::vars{IPMI_HOSTNAME}, '-U', $bmwqemu::vars{IPMI_USER}, '-P', $bmwqemu::vars{IPMI_PASSWORD});
+    my @cmd = ('ipmitool', split(' ', $ipmi_options), '-H', $bmwqemu::vars{IPMI_HOSTNAME}, '-U', $bmwqemu::vars{IPMI_USER}, '-P', $bmwqemu::vars{IPMI_PASSWORD});
     push(@cmd, split(/ /, $cmd));
```

Comment:
ok, that would fix the one specific issue. But I strongly suggest to not replicate this code from os-autoinst but rather use the functions from there directly and as required extend the backend.

Reply:
Good point, we'd better to call `ipmi_cmdline()` from `backend/ipmi.pm` here.